openconcept/atmospherics/true_airspeed.py

```diff
@@ -91,4 +90,4 @@ def compute(self, inputs, outputs):
     def compute_partials(self, inputs, partials):
         rho_isa_kgm3 = 1.225
```

Comment:
This should probably be moved to the new constants file.

Reply:
I didn't move it to the constants folder because density isn't really a constant and is only used in this component. I thought it'd make it MORE confusing because density is used as a variable all over the place